### PR TITLE
Document CVE-2016-6319

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -62,6 +63,15 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+
+#### <a id="2016-6319"></a>CVE-2016-6319:
```
Comment (domcleal) [inline security.md]: missing a title here

File: security.md [hunk 2]
```diff
@@ -62,6 +63,15 @@ All security advisories made for Foreman are listed below with their correspondi
...
+
+Label parameter of all form helpers was not escaped allowing XSS (cross-site scripting). The Foreman itself did not contain exploitable code but other plugins that relied on form helpers could be vulnerable. One known vulnerable plugin is Remote Execution. All plugin versions are affected.
```
Comment (domcleal) [inline security.md]: I'd change "All plugin versions" to say "All versions of this plugin".

File: security.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ The policy of the project is to treat all newly reported issues as private, and
 
 All security advisories made for Foreman are listed below with their corresponding [CVE identifier](http://cve.mitre.org/).
 
+* [CVE-2016-6319: Foreman form helpers do not escape JS when rendering label](security.html#2016-6319)
```
Comment (domcleal) [inline security.md]: Order numerically, between 6320/5390?

**Comment by ares** [comment]
thanks, all fixed, I've put the description under 6320 too

**Comment by domcleal** [comment]
Thanks @ares.

**Pull request merged**